### PR TITLE
Fix mistakes in docs for db

### PR DESCRIPTION
**Pull request by MykytaKyt**
# What does this PR do?

Removes mistakes or inconsistencies that appeared during the database creation. And also adds some information to README.md.

## Explanation

To fix mistakes and add new info.

### CODE REVIEW

File: docs/diagram.md [hunk 1]
```diff
@@ -72,13 +71,13 @@ var name              | type          | constraints
 --------------------- | ------------- | -------------------------------------------- | ------------------------------------------------
 **id**                | PK, INT       | NOT NULL                                     | stores id of a employee
 **equipment_type_id** | FK, INT       | NOT NULL                                     | refers to the table equipment_type
-**name**              | FK, CHAR(255) | NOT NULL                                     | stores the name of the equipment
+**name**              | CHAR(255) | NOT NULL                                     | stores the name of the equipment
```
Comment (aartiukh) [inline docs/diagram.md]: why did you remove `FK` here?

File: docs/diagram.md [hunk 2]
```diff
@@ -117,4 +116,4 @@ var name          | type    | constraints | description
 **employee_id**   | FK, INT | NOT NULL    | refers to the table employee
 **equipment_id**  | FK, INT | NOT NULL    | refers to the table equipment
 **date_of_issue** | DATE    | NOT NULL    | stores info when equipment was issued
-**day_of_return** | DATE    | NOT NULL    | stores info when equipment was returned
+**date_of_return** | DATE    | NULL | stores info when equipment was returned
```
Comment (aartiukh) [inline docs/diagram.md]: please fix indentations in all tables

**Review by aartiukh — CHANGES_REQUESTED** [review]

File: docs/diagram.md (hunk 1 above)
Comment (MykytaKyt) [reply docs/diagram.md]: Because the name is not `FK`

Comment (paulzcooper) [reply docs/diagram.md]: The field `name` should be `text`

**Review by aartiukh — DISMISSED** [review]

File: docs/diagram.md (hunk 1 above)
Comment (MykytaKyt) [reply docs/diagram.md]: @paulzcooper fixed

**Review by aartiukh — APPROVED** [review]

**Review by OleksandrZhytnyk — APPROVED** [review]

**Review by paulzcooper — APPROVED** [review]

**Review by aartiukh — CHANGES_REQUESTED** [review]
Why did you remove this file?
`docs/images/database.png`

**Comment by MykytaKyt** [comment]
> Why did you remove this file?
> `docs/images/database.png`

Because I replaced it with another database.png

**Review by aartiukh — APPROVED** [review]

**Pull request merged**